.travis-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Originally from https://gist.github.com/domenic/ec8b0fc8ab45f39403dd, with my
+# modifications for webpack.
+
+TARGET_BRANCH="pr3__bikeshed"
```

Comment:
Yeah, I wouldn't want you to have wasted all that time.  But yes, I think that having source on master and compiled output on gh-pages would be ideal.  Dom will need to manage the owner-related duties on that though.